object_manager.go

```diff
@@ -435,17 +489,22 @@ func (objMgr *ObjectManager) GetIpAddressFromHostRecord(host HostRecord) (string
 	return host.Ipv4Addrs[0].Ipv4Addr, err
 }
 
-func (objMgr *ObjectManager) UpdateHostRecordWithoutDNS(hostRref string, ipAddr string, macAddress string, vmID string) (string, error) {
+func (objMgr *ObjectManager) UpdateHostRecord(hostRref string, ipAddr string, macAddress string, vmID string, vmName string) (string, error) {
 
 	recordHostIpAddr := NewHostRecordIpv4Addr(HostRecordIpv4Addr{Mac: macAddress, Ipv4Addr: ipAddr})
 	recordHostIpAddrSlice := []HostRecordIpv4Addr{*recordHostIpAddr}
 	updateHostRecord := NewHostRecord(HostRecord{Ipv4Addrs: recordHostIpAddrSlice})
+
+	ea := objMgr.getBasicEA(true)
 	if vmID != "" {
-		ea := objMgr.getBasicEA(true)
 		ea["VM ID"] = vmID
 		updateHostRecord.Ea = ea
 	}
 
+	if vmName != "" {
+		ea["VM Name"] = vmName
+		updateHostRecord.Ea = ea
```

Comment:
Same comment as I've made for Fixed Address.

Reply:
same comment,  vmId was already present , i did add vmName just to make sure attributes of vm would get covered